testnet/stacks-node/src/tests/epoch_25.rs

```diff
@@ -169,17 +170,25 @@ fn microblocks_disabled() {
     submit_tx(&http_origin, &tx);
 
     // wait until just before epoch 2.5
-    loop {
+    wait_for(30, || {
```

Comment:
You might need a longer wait time here because it has to produce a few blocks to get to the epoch 2.5 transition. Like mayb 120 or even 180?